src/helpers.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * Reduce the code which written in Vue.js for getting the state.
+ * @param {String} namepsace
```

Comment:
Since `namespace` is an optional parameter, it should be `@param {String} [namespace]`. (same for other helpers)
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values